### DISCUSSION TITLE
Using nuxt3, the source code has json data problems

### DISCUSSION THREAD

**Discussion by wangrunze2023** [nuxt/nuxt#26080] [Questions]
### Describe the feature

How to remove json data in web page source code and keep only html? Please help me if you are an expert on this issue.
I hope this issue can be pointed out in the documentation

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by xjccc** [comment]
The feature

look at this https://github.com/nuxt/nuxt/issues/25336
